mappings/net/minecraft/server/world/ChunkIo.mapping

```diff
@@ -0,0 +1,5 @@
+CLASS net/minecraft/class_5318 net/minecraft/server/world/ChunkIo
```

Comment:
ChunkStorage, we discussed this on discord  (i told you not to mess with chunk classes but you still did)

Reply:
ok will yeet

Reply:
No just rename it to `ChunkStorage` , i will fix my clashes